docs/setup/with-docker.md

```diff
@@ -26,6 +26,31 @@ Now you can connect to the running server with any client. For a quick
 start, try [creating accounts and transfers in the Node
 CLI](../usage/node-cli).
 
+## `error: SystemResources` on macOS
+
+If you get `error: SystemResources` when running TigerBeetle in Docker
+on macOS, you will need to do one of the following:
+
+1. Run `docker run` with `--cap-add IPC_LOCK`
+1. Run `docker run` with `--ulimit memlock=-1:-1`
+1. Or modify the defaults in `$HOME/.docker/daemon.json` and restart the Docker for Mac application:
```

Comment:
```suggestion
2. Run `docker run` with `--ulimit memlock=-1:-1`
3. Or modify the defaults in `$HOME/.docker/daemon.json` and restart the Docker for Mac application:
```

For those of use who read through the text files :wink:.

Reply:
Done, thanks!